Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_cff.py

```diff
@@ -4,6 +4,7 @@
 ecalTrgHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
 #    eventSetupPathsKey='AlCa_EcalPhiSym*', # this is the HLT path that can be used                                                                                                          
      eventSetupPathsKey='EcalTrg',
+     throw = cms.bool( False ),
```

Comment:
@Martin-Grunewald 
to get some context : what is motivation behind this change and what benefits do you get?